lib/controllers/rocket.py

```diff
@@ -1,9 +1,10 @@
-from typing import Dict, Any, Union
-import jsonpickle
+from typing import Union
 import ast
+import jsonpickle
+
+from fastapi import HTTPException, status
 
-from fastapi import Response, status
-#from inspect import getsourcelines
+# from inspect import getsourcelines
```

Comment:
Can't you just simply delete this comment instead?

Reply:
It is a reminder that I need to handle cases where chute.trigger.__name__ == "<lambda>"

See line 524 of the same file:
```
if chute.trigger.__name__ == "<lambda>":
    # line = getsourcelines(chute.trigger)[0][0]
    # _parachute_ejection_signal_trigger[chute.name] = "Ejection signal trigger: " + line.split("lambda ")[1].split(",")[0].split("\n")[0]
    pass
```

But I'll add a TODO on top of it as well, thanks for pointing this out.